.github/workflows/static-analysis.yaml

```diff
@@ -83,18 +83,12 @@ jobs:
     runs-on: ${{ inputs.runsOn }}
     strategy:
       matrix:
-        workingdir: [extender, telemetry-aware-scheduling, gpu-aware-scheduling]
+        workingdir: [extender, telemetry-aware-scheduling]
```

Comment:
I think it is better to keep it any static analysis for gas

Reply:
Added the GAS module back for scanning